### PR TITLE
fix(test):fix clean of ocp is executed twice when INT e2e run

### PR DESCRIPTION
**Pull request by lihongyan1**
fix clean of ocp is executed twice when interrupt e2e run and some comments format

### CODE REVIEW

File: test/run-e2e-ocp.sh [hunk 1]
```diff
@@ -22,25 +22,26 @@ declare SHOW_USAGE=false
 
 cleanup() {
 	# NOTE: clears trap INT
-	trap - INT
+	info "Cleaning up ocp..."
 
 	# skip cleanup if user requested help
 	$SHOW_USAGE && return 0
 	delete_obo
+	return 1
```
Comment (sthaha) [inline test/run-e2e-ocp.sh:30]: why are we returning 1 ? `return 0` mean everything worked as normal. Any non-zero indicates error

File: test/run-e2e.sh [hunk 2]
```diff
@@ -28,6 +28,7 @@ cleanup() {
 	# shell check  ignore word splitting when using jobs -p
 	# shellcheck disable=SC2046
 	[[ -z "$(jobs -p)" ]] || kill $(jobs -p) || true
+	return 1
```
Comment (sthaha) [inline test/run-e2e.sh:31]: we should return 1 here as well.

File: test/run-e2e-ocp.sh [hunk 3]
```diff
@@ -22,25 +22,26 @@ declare SHOW_USAGE=false
 
 cleanup() {
 	# NOTE: clears trap INT
-	trap - INT
+	info "Cleaning up ocp..."
```
Comment (sthaha) [inline test/run-e2e-ocp.sh:25]: header instead of "info" :)

File: test/run-e2e-ocp.sh (hunk 1 above)
Comment (lihongyan1) [reply test/run-e2e-ocp.sh:30]: Usually we interrupt a program run when found sth abnormal

File: test/run-e2e.sh (hunk 2 above)
Comment (lihongyan1) [reply test/run-e2e.sh:31]: If we don't return 1 here, `run-e2e-ocp.sh` calls `run-e2e.sh` and suppose `run-e2e.sh` end normally and continue it's run and execute clean, the trap invoke clean function also

File: test/run-e2e-ocp.sh (hunk 3 above)
Comment (lihongyan1) [reply test/run-e2e-ocp.sh:25]: `cleanup()` of `run-e2e.sh` has `info`, suppose here should be consistent

**Comment by sthaha** [comment]
@lihongyan1 , can we close this one since #271 got merged?

**Pull request closed without merging**